flink-ai-flow/ai_flow/api/configuration.py

```diff
@@ -83,8 +83,8 @@ def get_default_project_config():
 
 def set_project_master_uri(master_uri: Text):
```

Comment:
Better to change function name and parameter to `server_uri`

Reply:
Fixed in dc55b6d